src/Console/GlobalCommunityBlock.php

```diff
@@ -52,19 +53,21 @@ protected function doExecute()
 			return 0;
 		}
 
-		if (count($this->args) > 1) {
+		if (count($this->args) > 2) {
 			throw new \Asika\SimpleConsole\CommandArgsException('Too many arguments');
 		}
 
 		if ($a->getMode()->isInstall()) {
 			throw new \RuntimeException('Database isn\'t ready or populated yet');
 		}
 
-		$contact_id = Contact::getIdForURL($this->getArgument(0));
-		if (!$contact_id) {
+		$contact_url = Contact::getIdForURL($this->getArgument(0));
```

Comment:
Why did you rename this value? It returns the Id for the URL, so it was right I guess

Reply:
You're right, I didn't see the `Contact::getIdForURL` call.